src/tools/tidy/src/deps.rs

```diff
@@ -672,9 +672,9 @@ fn check_license_exceptions(metadata: &Metadata, exceptions: &[(&str, &str)], ba
                 }
                 Some(pkg_license) => {
                     if pkg_license.as_str() != *license {
-                        println!("dependency exception `{name}` license has changed");
-                        println!("    previously `{license}` now `{pkg_license}`");
-                        println!("    update EXCEPTIONS for the new license");
+                        eprintln!("dependency exception `{name}` license has changed");
+                        eprintln!("    previously `{license}` now `{pkg_license}`");
+                        eprintln!("    update EXCEPTIONS for the new license");
```

Comment:
Suggestion: probably revert this because this is actual "info" output per dep checking?